packages/x-date-pickers/src/DesktopDatePicker/DesktopDatePicker.tsx

```diff
@@ -91,8 +92,12 @@ const DesktopDatePicker = React.forwardRef(function DesktopDatePicker<
     props,
     valueManager: singleItemValueManager,
     valueType: 'date',
-    getOpenDialogAriaText:
-      props.localeText?.openDatePickerDialogue ?? translations.openDatePickerDialogue,
+    getOpenDialogAriaText: buildGetOpenDialogAriaText({
+      utils,
+      formatKey: 'fullDate',
+      contextTranslation: translations.openDatePickerDialogue,
+      propsTranslation: translations.openDatePickerDialogue,
```

Comment:
Shouldn't this be? 🤔 
```suggestion
      propsTranslation: props.localeText?.openDatePickerDialogue,
```

Reply:
:grimacing: Definitely, thanks for catching it!